### PR TITLE
Add 3 lines utilities

### PR DESCRIPTION
**Pull request by pmbremner**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [ X] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [X ] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### CODE REVIEW

**Comment by pmbremner** [comment]
I didn't change the indentation that originally in the file, but that test failed. I will go back and re-indent and recommit.

File: source/utilities.cc [hunk 1]
```diff
@@ -1518,17 +1545,24 @@ namespace aspect
...
            * of off-diagonals.
            */
           void resize(int dim, int n_u, int n_l);
 
+
+
           /**
            * Return the dimension of the matrix
            */
           int dim() const;
```
Comment (bangerth) [inline source/utilities.cc:1562]: Ah, you couldn't know -- we only apply the three-line rule to global functions but not member functions :-)

**Review by bangerth — APPROVED** [review]
Yes, this is a strict improvement over the previous state!

**Comment by bangerth** [comment]
In hindsight, our rule really isn't particularly clear. We do the three lines (i) between declarations of global functions, (ii) between the implementation of member functions, but not (iii) the declaration of member functions within the class declaration.

Either way, I'm in favor of this!

File: source/utilities.cc [hunk 2]
```diff
@@ -1518,17 +1545,24 @@ namespace aspect
            * Constructor, see resize()
            */
           band_matrix(int dim, int n_u, int n_l);
+          
+          
+
```
Comment (gassmoeller) [inline source/utilities.cc:1550]:
```suggestion



```

File: source/utilities.cc [hunk 3]
```diff
@@ -1545,63 +1581,88 @@ namespace aspect
...
+
           /**
            * Writeable access to element A(i,j), indices going from
            * i=0,...,dim()-1
            */
           double &operator () (int i, int j);
+          
+          
+
```
Comment (gassmoeller) [inline source/utilities.cc:1593]:
```suggestion



```

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**